sapi/fpm/fpm/fpm_unix.c

```diff
@@ -349,6 +358,103 @@ static int fpm_unix_conf_wp(struct fpm_worker_pool_s *wp) /* {{{ */
 }
 /* }}} */
 
+#if HAVE_FPM_CPUAFFINITY
+struct fpm_cpuaffinity_conf {
+#if defined(HAVE_SCHED_SETAFFINITY) || defined(HAVE_CPUSET_SETAFFINITY)
+	cpu_set_t cset;
+#endif
+	long min;
```

Comment:
I find these slightly misleading, as they are only used as temporary values for `fpm_cpuaffinity_add`. I'd prefer if `min`/`max` were parameters to `fpm_cpuaffinity_add` but it's not a big deal.

Reply:
yeah agree, probably not much point to define them in struct as it can be just param. Also agree that it's not a big deal though :)

Reply:
Ok I will rephrase it. Please can you change this too.